c2corg_ui/templates/image/filters.html

```diff
@@ -0,0 +1,68 @@
+<%!
+from c2corg_common.attributes import default_langs, activities, image_categories, image_types
+%>
+<%namespace file="../helpers/filters.html" import="add_multiselect"/>
+
+<form app-search-filters app-search-filters-controller-name="appSearchFiltersController">
+  <div class="filters">
+    <div class="row simple-filters">
+
+      <div class="filter">
+        <label translate>date_taken</label><br>
```

Comment:
By the way I don't think the date is a "level 1" criteria for images. Not sure if the list of advanced search criteria is available anywhere.

Here is my suggestions for level 1 criterias (always displayed): 
* title
* categories
* activities